src/main/java/pl/wedding/management/service/weddinghall/model/WeddingHall.java

```diff
@@ -0,0 +1,23 @@
+package pl.wedding.management.service.weddinghall.model;
+
+import lombok.*;
+
+import java.util.Set;
+
+@Builder
+@Getter
+public class WeddingHall {
+    private final Set<TypesOfHalls> typesOfHalls;
+    private final Set<HallsEquipment> hallEquipment;
+    private final String owner;
+    private final int max_guests;
+    private final int numberBuilding;
+
+    public WeddingHall(Set<TypesOfHalls> typesOfHalls, Set<HallsEquipment> hallEquipment, String owner, int max_guests, int numberBuilding) {
```

Comment:
Why not lombok @RequiredArgsConstructor , you already use @Getter and @Builder

Reply:
jesli zamiast konstruktora dam @RequiredArgsConstructor podkresla mi pola na czerwono